alibi_detect/cd/pytorch/context_aware.py

```diff
@@ -4,13 +4,36 @@
 from typing import Callable, Dict, Optional, Tuple, Union
 from alibi_detect.cd.base import BaseContextMMDDrift
 from alibi_detect.utils.pytorch import get_device
-from alibi_detect.utils.pytorch.kernels import GaussianRBF
+from alibi_detect.utils.pytorch.kernels import BaseKernel, GaussianRBF
 from alibi_detect.cd._domain_clf import _SVCDomainClf
 from tqdm import tqdm
 
 logger = logging.getLogger(__name__)
 
 
+def _sigma_median_diag(x: torch.Tensor, y: torch.Tensor, dist: torch.Tensor) -> torch.Tensor:
```

Comment:
Not a huge fan of having this at the top of the script. Really I guess it belongs alongside the `sigma_median` function in utils anyway (perhaps even as an option within that function). Could be a good time to move (and then import) it.

Reply:
The new parameter implementation assigns the default value of 1 during __init__, so this function is no longer needed.